flink-python/pyflink/fn_execution/coder_impl_slow.py

```diff
@@ -114,7 +114,7 @@ class ValueCoderImpl(LengthPrefixBaseCoderImpl):
     def __init__(self, field_coder: 'FieldCoderImpl'):
         super(ValueCoderImpl, self).__init__(field_coder)
 
-    def encode_to_stream(self, value, out_stream):
+    def encode_to_stream(self, value, out_stream: OutputStream):
```

Comment:
What about add the type hint to all `encode_to_stream` and `decode_from_stream`?